src/stactools/nalcms/commands.py

```diff
@@ -101,4 +102,25 @@ def create_item_command(destination: str, source: str, region: str, gsd: str, ye
         else:
             print(f"{gsd}m_{year}_{region} not found in NALCMS")
 
+    @nalcms.command(
+        "create-cog",
+        short_help="Transform Geotiff to Cloud-Optimized Geotiff.",
+    )
+    @click.option("-d", "--destination", required=True, help="The output directory for the COG")
+    @click.option("-s", "--source", required=True, help="Path to an input GeoTiff")
+    def create_cog_command(destination: str, source: str) -> None:
+        """Generate a COG from a GeoTiff. The COG will be saved in the desination
+        with `_cog.tif` appended to the name.
+
+        Args:
+            destination (str): Local directory to save output COGs
+            source (str): An input NALCMS Landcover GeoTiff
+        """
+        if not os.path.isdir(destination):
+            raise IOError(f'Destination folder "{destination}" not found')
+
+        output_path = os.path.join(destination, os.path.basename(source)[:-4] + "_cog.tif")
+
+        cogify(source, output_path)
```

Comment:
We should add the extra gdal args where applicable. The[ `OVERVIEWS=IGNORE_EXISTING` ](https://github.com/stactools-packages/nalcms/pull/9/files#:~:text=cmd%20%3D%20%5B,%5D) means that overviews will be recreated each time. In the case of overviews being created before on the tifs, this will wipe them and recreate the overviews to align with COG spec.

Reply:
Does anyone know if the other settings that were previously specified are correct?

Reply:
Good point. The other settings look to be the default ones in the gdal driver.

Reply:
Included these, copied from `aafc_landuse`. They don't look to be the defaults, as per https://gdal.org/drivers/raster/cog.html

```python
        args = [
                "-co",
                "BLOCKSIZE=512",
                "-co",
                "compress=deflate",
                "-co",
                "predictor=yes",
                "-co",
                "OVERVIEWS=IGNORE_EXISTING"
        ]
```

Reply:
Cool, thanks.
I think deflate and predictor can go since the cog spec recommends that no compression be done. We should also try to minimize our touching of the data.
 - https://github.com/cogeotiff/cog-spec/blob/master/spec.md#unspecified-points

The `blocksize=512` is the default for the cog driver.
 - https://gdal.org/drivers/raster/cog.html#creation-options

Reply:
Okay, just left it at `OVERVIEWS=IGNORE_EXISTING`